### ISSUE TITLE
Modify "skip existing" behavior or add volume edit command?

### ISSUE THREAD

**Issue by scaleoutsean** [solidfire/solidfire-docker-driver#17]
From readme.md: "The Driver create method checks the SolidFire Cluster to see if the Volume already exists, if it does it just passes back the info for the existing volume, otherwise it runs through the create process and creates the Volume on the SolidFire cluster."

My question is for `solidfire-docker-driver volume edit`  - could that be added in a future version?
At the moment because of this "skip existing" behavior (which is reasonable) it's not possible to re-run the command with different params to set a larger size or (more often) different QoS, changes must be done externally which is not too much trouble, but one way to implement "change QoS" in the create method would be to apply `-o` options if the volume exists. It's uglier but probably easier... 

What do you think?

**Comment by j-griffith** [comment]
yeah... I sort of went back and forth on that.  Initially my philosophy was that if you wanted to update/modify a volume you would do it from the sfcli that's included as part of the driver.  The idea being that Docker itself didn't really have any real knowledge of the volume or it's characteristics.

In retrospect, there may be some value in allowing a special option to create that does the update via the Docker CLI.  I thought this would be a little confusing though, without a specific "update or modify" API.

**Comment by scaleoutsean** [comment]
> I thought this would be a little confusing though, without a specific "update or modify" API.

I agree. 
In terms of the effort, what's the difference?
- If Volume Name exists, pass just `-o` options using `VolumeModify`. Ignore new volume sizes. Undocumented feature in docker driver. 
- Add `volume modify` in `sfcli` (needs to support `totalSize`, etc., I guess?) as a fully documented feature in `sfcli`.

If the former is much easier, I'd be happy to have it because (80% of desired functionality with 20% of work compared to `sfcli` implementation?).
